go-controller/pkg/kubevirt/switch.go

```diff
@@ -0,0 +1,22 @@
+package kubevirt
+
+import (
+	"net"
+	"strings"
+
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
+)
+
+const (
+	ARPProxyIPv4 = "169.254.1.1"
+	ARPProxyIPv6 = "d7b:6b4d:7b25:d22f::1"
+)
+
+// ComposeARPProxyLSPOption returns the "arp_proxy" field needed at router type
+// LSP to implement stable default gw for pod ip migration, it consiste of
+// generated MAC address and a link local ipv4 and ipv6, it's the same
+// for all the logical switches.
+func ComposeARPProxyLSPOption() string {
+	mac := util.IPAddrToHWAddr(net.ParseIP(ARPProxyIPv4)).String()
```

Comment:
this is the same everytime you call it. Maybe we could define it as a string const ?

Reply:
You are very limited on what you can execute composing a const, also you can only compose this when the var `config.Default.ClusterSubnets` has content so it's better to have a function, an alternative would be to compose this at config time so it runs ones, but feels to me that is not worth it right now.

Reply:
I will add a constant for the MAC.